packages/uui-toggle/lib/uui-toggle.element.ts

```diff
@@ -24,6 +25,7 @@ import { css, html } from 'lit';
  *  @cssprop --uui-toggle-background-color-focus - Set the toggle background color when focused
  *  @extends UUIBooleanInputElement
  */
+@customElement('uui-toggle')
```

Comment:
This is using the @customElement decorator from Lit, I guess it should be using the @defineElement